pyrdp/ui/qt.py

```diff
@@ -151,7 +163,6 @@ def runOnMainThread(self, target: callable):
     @property
     def screen(self):
         return self._buffer
-
```

Comment:
Not sure what this is doing here given your taste to reindent and add new lines to things :wink:

Reply:
PEP8 :)

Reply:
![pep8-wat](https://user-images.githubusercontent.com/546325/79359606-ce907780-7f10-11ea-8d41-ec1a78997531.png)

No empty lines between a property and a method? This is PEP8?

Reply:
I missed this one :(

Reply:
wait

Reply:
i'm now ashamed of myself

Reply:
Don't be

Reply:
haha it was mostly a joke